R/learner_aorsf_surv_aorsf.R

```diff
@@ -173,7 +173,11 @@ LearnerSurvAorsf = R6Class("LearnerSurvAorsf",
         pred_type = "surv",
         .args = pv
       )
-      mlr3proba::.surv_return(times = utime, surv = surv)
+      diffs_time = c(utime[1], diff(utime))
+      response = apply(surv, MARGIN = 1, FUN = function(x){
+        sum(diffs_time * x)
+      })
+      mlr3proba::.surv_return(times = utime, surv = surv, response = response)
```

Comment:
If this could be done (ie calculating the `response` prediction) via a function in `aorsf` is that preferable @sebffischer?

Reply:
There is a non-exported function in `aorsf` for this called `rmst`. Let me know your preference on what to do. I can make it an exported function if that is important.

```r
rmst <- function(probs, times){

 diffs <- collapse::fdiff(times)
 diffs[1] <- times[1]
 collapse::fsum(diffs * probs)

}
```

Reply:
Ah if it's not exported then no need, it's a small thing anyway, thanks!